packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -338,6 +338,16 @@ module.exports = async ({
     }
   }
 
+  /**
+   * Loads pdf assets
+   */
+  rules.pdfs = () => {
+    return {
+      use: [loaders.file()],
+      test: /\.pdf$/,
```

Comment:
Yes I agree this was also something I was thinking about. How about `rules.miscFiles`?

Reply:
also this could get merged with existing rule that use `loader.file()` and be renamed to something more generic as @jquense mentioned:
https://github.com/gatsbyjs/gatsby/blob/0d9f892e5df04e7c38e3ca8cc32946d35e5dfc0c/packages/gatsby/src/utils/webpack-utils.js#L355-L360

Reply:
I'm all of a sudden running into some errors when running `npm run watch`. Here's a screenshot: 
![image](https://user-images.githubusercontent.com/29875483/43087521-133146d4-8e6e-11e8-9410-1e81763075ac.png)

Have you guys run into this error before and/or know how I can fix it?

Reply:
I occasionally get this on windows unfortunately. You can try deleting `gatsby/dist` and rerunning `npm run watch`